### ISSUE TITLE
[Feature Request] Ability to set up a board of my own

### ISSUE THREAD

**Issue by andbenn** [SecUSo/privacy-friendly-sudoku#120]
As a user, I may spot a Sudoku board while out and about.  I'd like to create a blank board and place the initial numbers down, then start playing.

Then complete the play as I normally would.

**Comment by coderPaddyS** [comment]
This feature already exists :D
Simply press on the circle instead of the stars for the difficulty rating.

**Comment by andbenn** [comment]
Yes, you are correct.  I completely and utterly missed that!  Thanks a ton, now I can pick up and take the daily board and other boards I discover with me.

Thank you and OK on closing, obviously.
